gauge_web_app_steps/app_context.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, ctx: ExecutionContext = None) -> None:
         self.image_path = ImagePath(config.get_browser().value, config.is_headless())
         self.images = Images(self.report)
         self.diff_formats = config.get_diff_formats()
+        self.explicit_timeout = None
```

Comment:
This should go into the scenario data_store.
Can you check, if the `self.implicit_timeout` can be removed, too?

Reply:
done